app/views/topics/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>New Topic</h1>
+
+<div class="row">
+  <div class="col-md-4">
+
+  </div>
+  <div class="col-md-8">
+    <%= form_for @topic do |f| %>
```

Comment:
the `topic_params` in the `topics_controller` appear to have a potential parameter `:user_id`, but I don't see it in the form.  when/where will that attribute get assigned?